Food_Tracker_App/app/src/main/res/values/strings.xml

```diff
@@ -3,4 +3,11 @@
     <string name="title_home">Home</string>
     <string name="title_dashboard">Dashboard</string>
     <string name="title_notifications">Notifications</string>
+    <string name="textIngredientDescription">Description</string>
+    <string name="textIngredientExpiry">Expiry Date</string>
+    <string name="textIngredientCategory">Category</string>
+    <string name="textIngredientQuantity">Quantity</string>
+    <string name="textIngredientAmount">Amount</string>
```

Comment:
would it be better to do the id names as snake_case and variable names as camelCase?